components/form/fields/TextField.tsx

```diff
@@ -7,13 +7,15 @@ import {
 import { setColor } from '../utilities'
 import { TextFieldProps } from '@/utils/types/components/form'
 
+const CHARACTER_COUNT = 600
 const TextField = ({ id, label, type, onChange, field }: TextFieldProps) => {
     return (
         <FormControl color={setColor(field)} error={Boolean(field.error)}>
             <MuiTextField
                 color={setColor(field)}
                 multiline={type === 'textarea'}
                 variant="standard"
+                inputProps={{ maxLength: CHARACTER_COUNT }}
```

Comment:
This should only be relevant if there is a flag passed into the component to denote to use it. Or, just on all textareas, in which you can just set the variable.

```
const isTextArea = type === 'textarea'`
```

And use as a check point everywhere related to the character count